packages/generator/src/edmx-to-vdm/v4/operation.ts

```diff
@@ -120,17 +120,31 @@ export function filterAndTransformOperations(
   operations: EdmxOperation[],
   isBound: boolean
 ): EdmxJoinedOperation[] {
-  const filteredByBoundOperations = operations.filter(
-    operation => (operation.IsBound.toLowerCase() === 'true') === isBound
+  const [withoutOperations, withOperations] = splitOperationsByMissingImports(
+    operationImports,
+    operations
   );
 
-  const [withoutOperation, withOperation] = splitMissingOperation(
-    operationImports,
-    filteredByBoundOperations
+  const relevantOperations = withOperations.filter(
+    operation => operation.IsBound === isBound
   );
 
+  const validBoundOperations =
+    validateBoundOperationParameters(relevantOperations);
```

Comment:
[req] The function is called `validateBoundOperationParameters()`. However it is also called for unbound operations. It works because the underlying `split...()` function ignores unbound operations, but I believe it should not even get to this and we should filter it out here already. That would be way more obvious. (see comment below)

```suggestion
  if (isBound) {
    const validBoundOperations = validateBoundOperationParameters(relevantOperations);
  }
```